backend/src/main/java/com/woowacourse/moragora/application/AttendanceService.java

```diff
@@ -109,10 +115,22 @@ public void updateAttendance(final Long meetingId,
     }
 
     public CoffeeStatsResponse countUsableCoffeeStack(final Long meetingId) {
-        final MeetingAttendances meetingAttendances = findMeetingAttendancesBy(meetingId);
-        validateEnoughTardyCountToDisable(meetingAttendances);
-        final Map<User, Long> userCoffeeStats = meetingAttendances.countUsableAttendancesPerUsers();
-        return CoffeeStatsResponse.from(userCoffeeStats);
+        validateMeetingExist(meetingId);
+        final List<Participant> participants = participantRepository.findByMeetingId(meetingId);
+        final int numberOfParticipants = participants.size();
+
+        final PageRequest pageRequest = PageRequest.ofSize(numberOfParticipants);
```

Comment:
```suggestion
        final PageRequest pageRequest = PageRequest.of(0, numberOfParticipants, Sort.by("event.date"));
```

이왕 Pageable을 사용한 김에, 정렬 조건으로 Sort를 활용하는 건 어떤가요?
Repository에 찾아가서 쿼리를 직접 읽지 않아도 정렬 조건을 파악 가능하다는 장점이 있을 것 같아요.

Reply:
Service에서 PageRequest를 사용하는 것이 Jpa에 의존적인 코드이기 때문에 사용하고 싶지 않았지만, 성능 개선때문에 사용하게 되었는데요. 그래서 service에서 최소한만 알고 있게 만들고 싶었어요. 미팅의 사람수는 미팅마다 달라지기 때문에 Service에서 지정을 해줘야하지만, 오름차순으로 정렬한다는 것은 동일하기 때문에 Service가 몰라도 된다고 생각을 했어요. 어떻게 생각하실까요?

Reply:
가장 오래된 지각 데이터부터 커피스택으로 사용한다는 것도 비즈니스 정책이라고 본다면 Service에서 지정해줘야 하는 부분이라고 볼 수 있을 것 같아요!

Reply:
포키 말도 이해가 되는 것 같아요! repository를 dao 관점으로 정말 db 관점으로 생각하면 포키의 말이 좋다고 생각이 드는데, repository를 도메인 관점으로 생각하면 이 정도의 비즈니스 정책은 알아도 될 것 같다는 생각이예요!  그리고 정렬 기준이 여러가지라면 포키의 말이 더 좋을 것 같은데 정렬 기준도 현재 1가지라서 이 메서드를 사용할 때마다 정렬기준을 지정하는 것이 불필요할 것 같기도 하고요.
 service에서는 커피스택으로 사용될 출석만 전달받으면 되는거고 안에 구현방식은 몰라도 될 것 같다는 생각이예요

Reply:
이해됐습니다! 말씀하신대로 현재는 정렬 기준이 하나로 고정되어 있으니 service가 데이터 관리에 대해 신경쓰지 않게 만드는 것도 장점이 있겠네요.